TFSIntegration/Script.ps1

```diff
@@ -0,0 +1,1609 @@
+
+        public static readonly String CONTROLLER_RESPONDED_WITH_ERRORS = "Controller responded with errors! Please check controller logs and try again! If does not help, try to restart controller.";
+        public static readonly String PLEASE_CONTACT_SUPPORT = "If nothing helps, please contact support https://leapwork.com/chat and provide the next information:  \n1.Plugin Logs  \n2.Leapwork and plugin version  \n3.Controller logs from the moment you've run the plugin.  \n4.Assets without videos if possible.  \nYou can find them {Path to Leaptest}/LEAPTEST/Assets  \nThank you";
+
+        public static readonly String ERROR_CODE_MESSAGE = "Code: {0} Status: {1}!";
+        public static readonly String COULD_NOT_CONNECT_TO = "Could not connect to {0}! Check it and try again! ";
+        public static readonly String COULD_NOT_CONNECT_TO_BUT_WAIT = "Could not connect to {0}! Check connection! The plugin is waiting for connection reestablishment! ";
+        public static readonly String CONNECTION_LOST = "Connection to controller is lost: {0}! Check connection! The plugin is waiting for connection reestablishment!";
+        public static readonly String INTERRUPTED_EXCEPTION = "Interrupted exception: {0}!";
+
+
+        public static readonly String STOPPING_RUN = "Stopping schedule {0} run {1}!";
+
+        public static readonly String STOP_RUN_FAIL = "Failed to stop schedule {0} run {1}!";
+
+        public static readonly String INVALID_ACCESS_KEY = "Invalid or empty access key!";
+
+        public static readonly String DATABASE_NOT_RESPONDING = "Data base is not responding!";
+
+        public static readonly String INVALID_VARIABLE_KEY_NAME = "Variable name is invalid or variable with such name is already in request!";
+
+        public static readonly String NO_DISK_SPACE = "No enough disk space to start schedule!";
+
+        public static readonly String PORT_NUMBER_IS_INVALID = "Port number is invalid, setting to default {0}";
+
+        public static readonly String TIME_DELAY_NUMBER_IS_INVALID = "Time delay number is invalid, setting to default {0}";
+
+
+        public static readonly String SCHEDULE_DISABLED = "Schedule {0}[{1}] is disabled!";
+
+        public static readonly String BUILD_SUCCEEDED = "SUCCEEDED";
+        public static readonly String BUILD_SUCCEEDED_WITH_ISSUES = "SUCCEEDED_WITH_ISSUES";
+        public static readonly String BUILD_FAILED = "FAILED";
+        public static readonly String BUILD_ABORTED = "ABORTED";
+
+        public static readonly String INVALID_BOOLEAN_TOKEN = "Failed to extract boolean token {0}, setting default value {1}";
+
+        public static readonly String FAILED_TO_PARSE_STRING_TO_GUID = "Failed to parse token {0} {1} to guid value";
+        public static readonly String STRING_TOKEN_NOT_FOUND = "Failed to find string token {0}";
+
+        public static readonly String INPUT_VALUES_MESSAGE = "LEAPWORK Plugin input parameters:";
```

Comment:
there is no LEAPWORK anymore, it is Leapwork now.

Reply:
Updated LEAPWORK to Leapwork in latest commit.